### PR TITLE
Add chinese translation

### PR DESCRIPTION
**Pull request by jaclarke**
Copied from fork: https://github.com/daichen-daisy/easy-edgedb

### CODE REVIEW

File: chapter0/index.md [hunk 1]
```diff
@@ -2,7 +2,12 @@
 leadImage: illustration_00.jpg
 ---
 
-# Chapter 0 - Easy EdgeDB
+# Welcome - Easy EdgeDB
+
+```{eval-rst}
+.. note::
+  See the `chinese translation </easy-edgedb/zh>`_ of this book
```
Comment (elprans) [inline chapter0/index.md:9]:
```suggestion
  This book is also available in `Chinese</easy-edgedb/zh>`_.
```

**Comment by 1st1** [comment]
I think China's 2 character code is "cn" (per ISO https://www.iban.com/country-codes) and per their national `.cn` domain.

**Comment by jaclarke** [comment]
Do we not want to use the language code here though?

**Comment by elprans** [comment]
No, it's an [ISO 639 language code](https://en.wikipedia.org/wiki/ISO_639-1), which is a different thing.  `zh` is correct.

**Comment by 1st1** [comment]
James, you can merge this if it's up to date and wouldn't cause deployment problems.

**Review by 1st1 — APPROVED** [review]

**Pull request merged**